BuntovSA/Lab_1/lababada.cpp

```diff
@@ -0,0 +1,79 @@
+﻿#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
+struct name
+{
+    int sum;
+    string io;
+};
+vector<string>sim;

```

Comment:
Please get rid of global variables which is usually a bad coding practice
Functions are designed for that purpose: you can return result from function instead of modifying global state and you can pass one function result to another